File: C/1144.c
// BEE 1144 - SEQUÊNCIA LÓGICA
#include <stdio.h>

int main()
{
    int n;
    int a, b, c, d, e, f;

    scanf("%d", &n);

    for (int i = 1; i <= n; i++)
    {
        a = i;
        b = a * a;
        c = a * b;
        d = i;
        e = b + 1;
        f = c + 1;
        printf("%d %d %d\n%d %d %d\n", a, b, c, d, e, f);
    }

    return 0;
}

File: PY/1144.py
# BEE 1144 - SEQUÊNCIA LÓGICA

n = int(input())

for i in range (1, n +1):
    a = i
    b = i * i
    c = b * i
    d = i
    e = b + 1
    f = c + 1
    
    print(f"{a} {b} {c}")
    print(f"{d} {e} {f}")